CarRentals/CarRentalsWebAPI/Interfaces/IBrandService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using CarRentals.Interfaces;
+using CarRentalsWebAPI.Models;
+
+namespace CarRentalsWebAPI.Services
+{
+    public interface IBrandService : IDataProcessing<Brand>
```

Comment:
I need to do this inheritance because I need to "drag" the methods create, delete, update and get from (NameClass)Service.
for example, If I remove this inheritance, BrandController wont recognize those methods from BrandService.

Also I can't get that Controller implements IBrandRepository directly for the same reason, I need to get the methods of service layer before repository layer

Reply:
I supposed that, but this is not the right way to do that. It's not about dragging but rather proxying those methods. You can have a separate interface or a service base class which implements those methods

Reply:
Ok, I will create an interface. Anyway the base interface will have the same statements that IDataProcessing, right?